[2025-01-17T21:10:52.268+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-17T21:10:52.282+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.extract_data manual__2025-01-17T21:10:49.200716+00:00 [queued]>
[2025-01-17T21:10:52.288+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.extract_data manual__2025-01-17T21:10:49.200716+00:00 [queued]>
[2025-01-17T21:10:52.289+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-17T21:10:52.300+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_data> on 2025-01-17 21:10:49.200716+00:00
[2025-01-17T21:10:52.305+0000] {standard_task_runner.py:72} INFO - Started process 167 to run task
[2025-01-17T21:10:52.308+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'extract_data', 'manual__2025-01-17T21:10:49.200716+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp2_7fvus_']
[2025-01-17T21:10:52.310+0000] {standard_task_runner.py:105} INFO - Job 36: Subtask extract_data
[2025-01-17T21:10:52.327+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-01-17T21:10:52.361+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.extract_data manual__2025-01-17T21:10:49.200716+00:00 [running]> on host 9fe2ea257df4
[2025-01-17T21:10:52.437+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-17T21:10:49.200716+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-17T21:10:49.200716+00:00'
[2025-01-17T21:10:52.438+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-17T21:10:52.946+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-17T21:10:52.956+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-17T21:10:52.957+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=extract_data, run_id=manual__2025-01-17T21:10:49.200716+00:00, execution_date=20250117T211049, start_date=20250117T211052, end_date=20250117T211052
[2025-01-17T21:10:53.003+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-17T21:10:53.025+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-17T21:10:53.028+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
